### DISCUSSION TITLE
Welcome to Photosynthesis Discussions!

### DISCUSSION THREAD

**Discussion by Zuiron** [Zuiron/Photosynthesis#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lemontea0075** [comment]
hi can you pls make the 1.19.2 working in modrinth?(if possible)

**Reply by Zuiron** [reply]
not possible

**Comment by yarnwoven** [comment]
will you be adding 1.21.4 anytime soon? really excited to try this mod out but was bummed i couldnt find a more recent version!!

**Reply by Zuiron** [reply]
no sorry.
